Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonComboBox.cs

```diff
@@ -2442,10 +2442,11 @@ protected override void OnPaint(PaintEventArgs? e)
                 ForceControlLayout();
             }
 
-            if (StateCommon.ComboBox.Content.SynchronizeDropDownWidth)
-            {
-                DropDownWidth = Size.Width;
-            }
+            // ToDo: Create a new API for this in a later version
```

Comment:
Why, Where did the requirement for this come from?

Reply:
Through some testing, I noticed that the width of the KCB didn't affect the dropdown, which leads to a poor user experience.

Reply:
That's a bug in the KryptonComboBox on the Form (Regression!!!!)
BUT, the Ribbon works as expected.
![image](https://github.com/user-attachments/assets/ddc01796-a163-4100-8817-09ac8157e029)

Reply:
Would be useful to have a option to link  up the size width & drop-down width.

Reply:
#1706